intelmq/bots/outputs/file/output.py

```diff
@@ -14,12 +15,21 @@ def init(self):
 
     def process(self):
         event = self.receive_message()
+        
 
         event_data = event.to_json(hierarchical=self.parameters.hierarchical_output)
-        self.file.write(event_data)
-        self.file.write("\n")
-        self.file.flush()
-        self.acknowledge_message()
+
+        try:
+            os.stat(self.parameters.file)
```

Comment:
If the file has been deleted between the `os.stat()` and the write here, this will fail again. I think it is better to simply try to write.